doc/src/manual/strings.md

```diff
@@ -839,12 +842,33 @@ julia> b"DATA\xff\u2200"
 
 The ASCII string "DATA" corresponds to the bytes 68, 65, 84, 65. `\xff` produces the single byte 255.
 The Unicode escape `\u2200` is encoded in UTF-8 as the three bytes 226, 136, 128. Note that the
-resulting byte array does not correspond to a valid UTF-8 string -- if you try to use this as
-a regular string literal, you will get a syntax error:
+resulting byte array does not correspond to a valid UTF-8 string :
+
+```jldoctest
+julia> isvalid("DATA\xff\u2200")
+false
+```
+
+As it was mentioned `CodeUnits{UInt8,String}` type behaves like read only array of `UInt8` and if you need a standard vector you can convert it using `Vector{UInt8}`:
 
 ```julia-repl
-julia> "DATA\xff\u2200"
-ERROR: syntax: invalid UTF-8 sequence
+julia> x = b"123"
```

Comment:
Any reason this is can't be a ```` ```jldoctest ```` ?

Reply:
I am truncating stack trace after error in the output and I thought in this case `julia-repl` should be retained.

Reply:
You can add `[...]`:
```
julia> x[1] = 0x32
ERROR: indexing not defined for Base.CodeUnits{UInt8,String}
[...]
```

Reply:
Fixed. Thanks!